WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m

```diff
@@ -173,35 +196,39 @@ - (void)configureBlogDetailHeader
                                                                  multiplier:1.0
                                                                    constant:headerWidth]];
         // Center the headerView inside the wrapper
-        [headerWrapper addConstraint:[NSLayoutConstraint constraintWithItem:self.headerView
+        [constraints addObject:[NSLayoutConstraint constraintWithItem:self.headerView
```

Comment:
(Absolutely no need to change anything!!). Just wanted to say... i recently discovered the `topAnchor` / `bottomAnchor` iOS 9 properties. Might become handy anytime else (you need way less code!)